README.md

```diff
@@ -139,14 +141,14 @@ If you run this tool from a normal directory it will check all files which are t
 The following output formats are supported:
 
 - **default**: Plain text, human readable output.<br/>
-      ```text
-      <file>:
-        <startingLine>-<endLine>: <message>
-      ```
+  `text
+    <file>:
+      <startingLine>-<endLine>: <message>
+    `
```

Comment:
This codeblock somehow got broken - it truly is a multiline output.

Reply:
Oops, I've fixed it, and also fixed the "Code Climate" format, so that it has syntax highlighting fo JSON.

Reply:
Nice, wanted to suggest it, but didn't want to mix that in with this one necessarily!